web/modules/cm-lang-ftml/src/data/data.ts

```diff
@@ -0,0 +1,67 @@
+import blocksTOML from "@root/ftml/conf/blocks.toml"
```

Comment:
Interesting you can do this

Reply:
It's because I whipped up a Vite plugin a while back for it. It's very nice to be able to just import random assets and it just works lol